thoth/storages/graph/postgres.py

```diff
@@ -3771,6 +3771,70 @@ def create_python_cve_record(
 
                 return False
 
+    def update_missing_flag_package_version(
+        self,
+        value: bool,
+        index_url: str,
+        package_name: str,
+        package_version: str
```

Comment:
Positional arguments in this module are in the following order:

1. package name
2. package version
3. index url

Please move the `value` parameter at the end so it reads as "what needs to be updated with what".